recipe/meta.yaml

```diff
@@ -14,16 +14,19 @@ build:
 
 requirements:
     build:
+      - compiler('c')
```

Comment:
This needs to be `{{ compiler('c') }}`

Reply:
`- {{ compiler('c') }}`

Reply:
@isuruf Thank you. I missed that in the rush. Committed the fix.